Units/parser-typescript.r/ts-class-fq-white.d/args.ctags

```diff
@@ -0,0 +1,3 @@
+--sort=no
+--typescript-kinds=fcigemnzpvCgGal
```

Comment:
You can use a wildcard character here (if you want):
```
--typescript-kinds=*
```
--<LANG>-kinds option is obsoleted.
Use --kinds-<LANG> instead.
So the option can be written as:
```
--kinds-typescript=*
```

Reply:
OK, I'll change it.

Reply:
Done.